hogben/tests/test_fisher.py

```diff
@@ -8,7 +8,7 @@
 import refnx
 import refl1d.experiment
 
-from hogben.simulate import simulate
+from hogben.simulate import SimulateReflectivity
```

Comment:
```suggestion
```
The `simulate` import was used to create a `refl1d` model, which was annoying enough that it made sense to do this from the `simulate` method rather than to repeat all that logic here. But that is no longer necessary.